target/linux/generic/hack-5.15/710-net-dsa-mv88e6xxx-default-VID-1.patch

```diff
@@ -0,0 +1,18 @@
+--- a/drivers/net/dsa/mv88e6xxx/chip.c
```

Comment:
This can be removed, as the upstream uses a different approach to solve mv88e6xxx's CPU port learning.

Reply:
Is this something that we can backport instead of forcing VLAN filtering and VLAN1?

Reply:
It's the "Bridge TX offload" feature, which was a large patchset. Is it worth it to backport this to 5.10?

Reply:
Mhhh it depends... Do we really need to add other patch to maintain ?